### DISCUSSION TITLE
HOSTEdit won't save on windows 11 pro latest updates

### DISCUSSION THREAD

**Discussion by PapaBravo57** [community/community#39708] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

As  I said HOSTEdit won't save on windows 11 pro latest updates  at least not to the etc folder I have taken ownership of the folder thinking that was the cause but no.

**Comment by ossanna16** [comment]
Hi there @PapaBravo57 and thank you for reporting this bug! Can you please add a bit more detail so we can understand the problem better?

**Comment by PapaBravo57** [comment]
If I add 127.0.0.1 and the url and click save it won't save
